Procfile

```diff
@@ -1 +1 @@
-web: gunicorn run:app
+web: gunicorn --worker-class eventlet -w 1 run:app
```

Comment:
we'll see if this work, this command is listed in the docs when deploying a flask app with websockets handled through `eventlet` 

https://flask-socketio.readthedocs.io/en/latest/